src/main/kotlin/net/maizegenetics/phgv2/utils/VariantLoadingUtils.kt

```diff
@@ -247,7 +253,7 @@ fun createSNPVC(assemblyTaxon: String, startPosition: Position, endPosition: Pos
 
     //Need to add AD for Alt >0 here so that the API will work correctly.  Otherwise it is treated as missing as it thinks AD = 0,0.
     // When coming from an assembly it should always use the ALT in a SNP pos
-    val gt = GenotypeBuilder().name(assemblyTaxon).alleles(Arrays.asList(altCall)).DP(2).AD(intArrayOf(0, 2, 0)).make()
+    val gt = GenotypeBuilder().name(assemblyTaxon).alleles(Arrays.asList(altCall)).DP(30).AD(intArrayOf(0, 2, 0)).make()
```

Comment:
You should probably change the AD to be 0,30,0

Reply:
This is set to 30 to match BioKotlin's TotalDepth value (Biokotlin MafToGVCF:convertVariantInfoToContext() ) which is used to create the VariantContext records for the gvcfs.  I think we should be consistent since the original gvcfs were created from BioKotlin and we are pulling those records.

Reply:
I read that wrong - yes, the AD should be 0,30,0 - I will change.